my-project/src/Controller/firstPage.php

```diff
@@ -22,7 +22,7 @@ public function date()
     {
```

Comment:
should this file be commited in this pr?
change kubek.jpg to use proper path for images

Reply:
if this is just spacing lets remove this change